sympy/physics/units/definitions.py

```diff
@@ -61,6 +61,8 @@
 
 optical_power = dioptre = D = Quantity("dioptre", 1/length, 1/meter)
 lux = lx = Quantity("lux", luminous_intensity/length**2, steradian*candela/meter**2)
+katal = kat = Quantity("katal", amount_of_substance/time, mol/second)
```

Comment:
Can you write a comment for `gray` saying "Absorbed dose of ionizing radiation" similarly for `katal`

Reply:
Ok, I will be on it ASAP.